mubench.pipeline/tasks/implementations/dataset_check_misuse.py

```diff
@@ -125,8 +125,12 @@ def _check_misuse_location_exists(self, project: Project, version: ProjectVersio
                         'Skipping location check for "{}": requires checkout of "{}".'.format(
                             misuse.id, version.id))
                 else:
-                    source_base_path = join(checkout.checkout_dir, version.source_dir)
-                    if not self._location_exists(source_base_path, location.file, location.method):
+                    location_exists = False
+                    for source_dir in version.source_dirs:
```

Comment:
It seems we have this logic (finding a file in one of the sources/classes directories) in multiple places. Can we centralize and reuse it?